src/box/lua/schema.lua

```diff
@@ -717,10 +741,10 @@ box.schema.space.create = function(name, options)
     local options_defaults = {
         engine = 'memtx',
         field_count = 0,
-        temporary = false,
     }
     check_param_table(options, options_template)
     options = update_param_table(options, options_defaults)
+    check_space_type(options.type)
```

Comment:
Writing again just to be sure: please move all the code that introduces space type to the first commit.
The last (the main) commit should just add one more type and a pair of `is_meta_temporary` helpers.

Reply:
done